env.default

```diff
@@ -75,3 +75,6 @@ USE_CLOUDINARY=False
 CLOUDINARY_CLOUD_NAME=
 CLOUDINARY_API_KEY=
 CLOUDINARY_API_SECRET=
+
+# Django config overrides
+DATA_UPLOAD_MAX_NUMBER_FIELDS=2500
```

Comment:
Let's leave this off. We don't want to make this a default thing - for testing, we can manually add this to our `.env` file during investigation instead.